ansible/k8s-kubespray-vae.yml

```diff
@@ -0,0 +1,85 @@
+# Kubespray includes a playbook with it's own set of groups.
```

Comment:
Again, would prefer if we're not cut-n-pasting this.

Reply:
kubespray requires fixed group names. If we're using our openstack dynamic inventory then applying the required kubespray names directly to the instances means it's impossible to distinguish between different idr-environments. This file is a workaround for dynamically converting our `idrenv-groupname` into `groupname` required by kubespray.